modules/tensor_mechanics/test/tests/umat/tests

```diff
@@ -0,0 +1,33 @@
+[Tests]
+  issues = '#14974'
+  design = 'AbaqusUmatMaterial.md'
+
+  [linear_strain_hardening]
+    type = 'Exodiff'
+    input = 'linear_strain_hardening.i'
+    exodiff = 'linear_strain_hardening_out.e'
+    library_mode = 'DYNAMIC'
+    valgrind = 'NONE'
+    requirement = 'The system shall provide an interface to use Abaqus UMAT materials as '
+                  'constitutive models, with support for stateful materials'
+  []
+
+  [elastic]
+    type = 'Exodiff'
+    input = 'elastic.i'
+    cli_args = "Materials/active='umat'"
+    exodiff = 'elastic_out.e'
+    library_mode = 'DYNAMIC'
+    valgrind = 'NONE'
+    requirement = 'The system shall provide an interface to use Abaqus UMAT materials as '
+                  'constitutive models, with support simple elastic materials models'
```

Comment:
'with support simple' -> 'with support for finite strain'

materials -> material